backend/util/auth.js

```diff
@@ -0,0 +1,22 @@
+const jwt = require('jsonwebtoken');
+
+module.exports = function authenticateJWT (req, res, next) {
```

Comment:
This file is not being imported anywhere. 🤔

Reply:
This will be imported in place where auth is required such as deleting or creating notice, events etc